packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -137,7 +137,9 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
 
   const verifyCode = useVerifyCode({
     onSuccess: () => {
-      verifyEmail.setVerifiedEmail(bookerForm.formEmail);
+      if (bookerForm.formEmail) return;
```

Comment:
Shouldn't this be..
```suggestion
      if (!bookerForm.formEmail) return;
```

?

Reply:
oh right.